test/transmuxer.test.js

```diff
@@ -2515,64 +2511,6 @@ QUnit.test('pipeline dynamically configures itself based on input', function() {
   QUnit.equal(transmuxer.transmuxPipeline_.type, 'aac', 'detected AAC file data');
 });
 
-QUnit.test('reuses audio track object when the pipeline reconfigures itself', function() {
```

Comment:
Now that the timing information (specifically, baseMediaDecodeTime) is going to be handled at a higher level, I am working on adding the test into media-sources.